### DISCUSSION TITLE
如何在 Linux 中下载 AList 共享的文件

### DISCUSSION THREAD

**Discussion by DustinWin** [AlistGo/alist#5578] [Q&A]
请问一下，在 Linux 中如何下载 AList 共享的文件（比如下载地址：https://alist.atri.ink/test.zip?sign=${sign}=:0 ），关键是如何获取 `sign` 值，不会整😭  
![photo_2023-11-26_13-30-43](https://github.com/alist-org/alist/assets/45238096/dc79f7be-01f1-4be1-ab4d-f1b5eb0cf7d0)

**Comment by BlueSkyXN** [comment]
打开guest，直接wget就行了，看你都这样了，还是简单点好

**Comment by DustinWin** [comment]
> 打开guest，直接wget就行了，看你都这样了，还是简单点好

我想要的是可以随时让下载脚本自动去下载 AList 共享的文件，所以必须获取到 `sign` 值

**Reply by BlueSkyXN** [reply]
把签名关掉就行了

**Comment by DustinWin** [comment]
> 把签名关掉就行了

是别人分享出来的文件哦。
